test-case/multiple-pipeline.sh

```diff
@@ -69,7 +69,7 @@ logger_disabled || func_lib_start_log_collect
 
 max_count=0
 # this line will help to get $PIPELINE_COUNT
-func_pipeline_export "$tplg" "type:any"
+func_pipeline_export "$tplg" "type:any & ~pcm:Amplifier Reference"
```

Comment:
Thanks @miRoox for the quick fix! Can you please go one small step further and remove the duplication to make sure it does not happen again?
```suggestion
MULTI_PIPELINE_FILTER='~pcm:Amplifier Reference'
func_pipeline_export "$tplg" "type:any & ${MULTI_PIPELINE_FILTER}"
```

+ use the same variable everywhere below.

Reply:
done